src/boardgameprojectfile/boardgameprojectfile.cpp

```diff
@@ -0,0 +1,55 @@
+#include "boardgameprojectfile.h"
+#include <QFile>
+#include <QDataStream>
+#include <QString>
+#include <QDebug>
+
+BoardGameProjectFile::BoardGameProjectFile()
+{
+
+}
+
+std::unique_ptr<BoardGame> BoardGameProjectFile::load(const QString &path)
+{
+

```

Comment:
Leve //TODO comment, that this method is intentionally empty and will be implemented later on